amplify/__tests__/functions/send-email/__snapshots__/email-template.test.ts.snap

```diff
@@ -0,0 +1,186 @@
+          q2hk+ZuE9DBnKBBsAJDYCB8BAIINAAg2ACDYAIBgAwCCDQDBBgAEGwAQbABQn9HdE2OX+RgANMmW
+          a9dYUQWApigAEGwAQLABAMEGAAQbAIINAAg2ACDYAIBgAwCCDQDBBgAEGwAQbABAsAEAwQaAYAMA
+          gg0ACDYAINgAgGADAIINAMEGAAQbABBsAECwAQDBBoBgAwCCDQAINgAg2ACAYANAsAEAwQYABBsA
+          EGwAQLABINgAgGADAIINAAg2ACDYAIBgA0CwAQDBBgAEGwAQbABAsAEg2ACAYAMAgg0ACDYAINgA
+          EGwAQLABAMEGAAQbABBsAAg2ACDYAIBgAwCCDQAINgAg2AAQbABAsAEAwQYABBsAEGwACDYAINgA
+          gGADAIINAAg2AAQbABBsAECwAQDBBgAEGwCCDQAINgAg2ACAYAMAgg0ACDYABBsAEGwAQLABAMEG
+          AAQbAIINAAg2AHDhfwIMAJCNcOrSAj7MAAAAAElFTkSuQmCC" alt="NHS Logo" width="90" />
+              </th>
+              <td>
+                <p style="margin-left: 12px; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px;">Notify</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p style="margin: 10px 0 30px 0; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px;">You have
+                  successfully submitted your template to NHS Notify.</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p style="margin: 0 0 5px 0; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px; font-weight: bold;">Template name</p>
+                <p style="margin: 0 0 30px 0; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px;">test-name</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p style="margin: 0 0 5px 0; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px; font-weight: bold;">Template ID</p>
+                <p style="margin: 0 0 30px 0; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px;">test-id</p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <p style="margin: 0 0 5px 0; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px; font-weight: bold;">Template content
+                </p>
+                <p style="margin: 0 0 30px 0; color: #757777; font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px;">
+                  This content does not have your formatting applied. This is not how your message will look when it's sent to
+                  recipients.
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2" style="background-color: #e8ecee; padding: 24px;">
+                <p style="font-family: 'Frutiger W01', Arial, Sans-serif; font-size: 16px; margin-top: 0; white-space: pre-wrap"># Message heading<br /><br />## Heading 2<br /><br />**bold text**<br /><br />*This text will be italic*<br /><br />* bullet<br />* bullet<br />* bullet<br /><br />[link text](https://www.nhs.uk/)<br /><br />### Ordered<br />1. Item 1<br />2. Item 2<br />3. Item 3<br />   1. Item 3a<br />   2. Item 3b</p>
```

Comment:
Is the markdown generating the expected html? For example I was expecting `**bold text**` to be `<b>bold text</b>`

Reply:
Yes, the idea is that the email shows the content as it was entered, not rendered although I appreciate that this is quite confusing!